Library/Formula/libfreenect2.rb

```diff
@@ -0,0 +1,143 @@
+require "formula"
+
+class Libfreenect2 < Formula
+  desc "Drivers/Example for Kinect V2"
+  homepage "https://github.com/OpenKinect/libfreenect2"
+  url "https://github.com/OpenKinect/libfreenect2/archive/master.zip"
```

Comment:
We'll need an actual release. The `master.zip` will update & break the checksum every time someone pushes a new commit there. Suggest you ask upstream if they're willing to tag a release for you here.